### PR TITLE
Bionano update: modify outputs and include script cut sites

### PR DESCRIPTION
**Pull request by gallardoalba**

### CODE REVIEW

**Comment by bgruening** [comment]
Cris this only works if we push the new image.

File: tools/bionano/bionano_scaffold.xml [hunk 1]
```diff
@@ -415,16 +423,22 @@
...
     </inputs>
     <outputs>
-        <data name="ngs_contigs_scaffold_fasta" format="fasta" from_work_dir="hybrid_scaffolds/bionano_bppAdjust_cmap_ngs_fasta_NGScontigs_HYBRID_SCAFFOLD.fasta" label="${tool.name} on ${on_string}: NGScontigs hybrid scaffold (fasta)"/>
-        <data name="ngs_contigs_scaffold_agp" format="txt" from_work_dir="hybrid_scaffolds/bionano_bppAdjust_cmap_ngs_fasta_NGScontigs_HYBRID_SCAFFOLD.agp" label="${tool.name} on ${on_string}: NGScontigs hybrid scaffold (agp)"/>
-        <data name="ngs_contigs_scaffold_gap" format="txt" from_work_dir="hybrid_scaffolds/bionano_bppAdjust_cmap_ngs_fasta_NGScontigs_HYBRID_SCAFFOLD.gap" label="${tool.name} on ${on_string}: NGScontigs hybrid scaffold (gap)"/>
+        <data name="ngs_contigs" format="fasta" from_work_dir="total_contigs_raw.fasta" label="${tool.name} on ${on_string}: NGScontigs">
+            <filter>trim_cut_sites == False</filter>
+        </data>
+        <data name="ngs_contigs_trimmed" format="fasta" from_work_dir="total_contigs_trimmed.fasta" label="${tool.name} on ${on_string}: NGScontigs trimmed">
+            <filter>trim_cut_sites == True</filter>
```
Comment (bgruening) [inline tools/bionano/bionano_scaffold.xml:430]: I think you can use `<filter>trim_cut_sites</filter>`

File: tools/bionano/bionano_scaffold.xml [hunk 2]
```diff
@@ -415,16 +423,22 @@
         -->
     </inputs>
     <outputs>
-        <data name="ngs_contigs_scaffold_fasta" format="fasta" from_work_dir="hybrid_scaffolds/bionano_bppAdjust_cmap_ngs_fasta_NGScontigs_HYBRID_SCAFFOLD.fasta" label="${tool.name} on ${on_string}: NGScontigs hybrid scaffold (fasta)"/>
-        <data name="ngs_contigs_scaffold_agp" format="txt" from_work_dir="hybrid_scaffolds/bionano_bppAdjust_cmap_ngs_fasta_NGScontigs_HYBRID_SCAFFOLD.agp" label="${tool.name} on ${on_string}: NGScontigs hybrid scaffold (agp)"/>
-        <data name="ngs_contigs_scaffold_gap" format="txt" from_work_dir="hybrid_scaffolds/bionano_bppAdjust_cmap_ngs_fasta_NGScontigs_HYBRID_SCAFFOLD.gap" label="${tool.name} on ${on_string}: NGScontigs hybrid scaffold (gap)"/>
+        <data name="ngs_contigs" format="fasta" from_work_dir="total_contigs_raw.fasta" label="${tool.name} on ${on_string}: NGScontigs">
+            <filter>trim_cut_sites == False</filter>
```
Comment (bgruening) [inline tools/bionano/bionano_scaffold.xml:427]: ` <filter>not trim_cut_sites</filter>`

File: tools/bionano/bionano_scaffold.xml (hunk 1 above)
Comment (gallardoalba) [reply tools/bionano/bionano_scaffold.xml]: Yep

**Review by bgruening — APPROVED** [review]

**Pull request merged**